docs/userguide/modules/getting-started/partials/connect-petclinic-postgresql.adoc

```diff
@@ -0,0 +1,114 @@
+application\'s admin needs to extract all the configuration details and
+create a Secret resource and expose it to the application through volume
+mount in Kubernetes. The steps would be something like this:
+
+. Identify the required values for connecting the application to the
+database
+. Locate the resources where the values are present
+. Take the values from different resources and create a Secret resource
+. Mount the Secret resource into the application
+. Depending on the application requirement the values should be exposed
+as env var or file.
+
+In this quick start, we are going to leverage {servicebinding-title} as a way
+to easily and safely connect the application to the database service. In
+order to do that, we\'ll need to create a `ServiceBinding` resource which
+will trigger the {servicebinding-title} to project the binding data
+into the application.
+
+=== Create the `ServiceBinding` resource to project the binding data:
+
+The `.spec` has two sections. The first is a list of service
+resources (`.spec.services`), and the second one is the application
+(`.spec.application`). The services resources points to the database\'s
+service resources. How the values are exposed from service resources are
+explained xref:exposing-binding-data:intro-expose-binding.adoc[Exposing Binding Data]
+section. The application points to a `Deployment`
+or any resource that is PodSpec compatible.
+
++++ <details><summary> +++
+`ServiceBinding` resource to project the binding data 
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$petclinic-postgresql-binding.yaml[tag=service-binding]
+----
++++ </div></details> +++
+
+For simplicity, you can run the following command in shell to create the
+resource:
+
+[source,bash,subs="attributes,macros"]
+----
+kubectl apply -f link:{attachmentsdir}/petclinic-postgresql-binding.yaml[pass:a[{site-url}/{page-component-name}/{page-module}/{attachmentsdir}/petclinic-postgresql-binding.yaml]] -n {quickstart-namespace}
+----
+
+To check if the binding was successful you can check the binding
+resource status conditions by running the following command:
+
+[source,bash,subs="attributes,macros"]
+----
+kubectl get -f link:{attachmentsdir}/petclinic-postgresql-binding.yaml[pass:a[{site-url}/{page-component-name}/{page-module}/{attachmentsdir}/petclinic-postgresql-binding.yaml]] -n {quickstart-namespace} -o jsonpath-as-json='{.status.conditions}'
+----
+
+You should see the output similar to the following:
+[source,json]
+----
+[
+  [
+    {
+      "lastTransitionTime": "2021-09-06T13:42:28Z",
+      "message": "",
+      "reason": "DataCollected",
+      "status": "True",
+      "type": "CollectionReady",
+    },
+    {
+      "lastTransitionTime": "2021-09-06T13:42:28Z",
+      "message": "",
+      "reason": "ApplicationUpdated",
+      "status": "True",
+      "type": "InjectionReady",
+    },
+    {
+      "lastTransitionTime": "2021-09-06T13:42:28Z",
```

Comment:
meanwhile service binding CRD is configured with custom print column, so that even simple

```
kubectl get servicebindings 
```

returns the status. Let's replace this big block with that.

Reply:
good point, thx